### ISSUE TITLE
Is it possible to work with multiple markers?

### ISSUE THREAD

**Issue by bzmelotti** [atayahmet/jquery.easypin#6]
We are using this plugin, but we are not sure if it is possible to use it for multiple markers. Is it possible?

**Comment by Bojianye** [comment]
Yes, It can support multiple markers. But it has some bugs about the marker's coordinate and some other functions.
